tools/lib-alert-tree/lib_alert_tree/models.py

```diff
@@ -112,10 +117,26 @@ def build_query(self, children, group_by=None):
         elif self == self.ANY:
             join_by = " or "
 
-        prefix = f"sum by ({', '.join(group_by)}) " if group_by else ""
+        prefix = f"sum by ({', '.join(group_by)}) " if group_by else "sum"
         children_query = join_by.join(child.query for child in children)
         return f"{prefix}({children_query}) >= 1"
 
+    @staticmethod
+    def build_json_path(children, group_by=None):
+        """Build a JSON Path to retrieve alert children."""
+        json_path_filters = " || ".join(map(lambda child: child.json_path, children))
```

Comment:
`map(lambda x: ..., iterable)` is not idiomatic Python, we usually prefer to use generator expressions:

```suggestion
        json_path_filters = " || ".join(child.json_path for child in children)
```